### ISSUE TITLE
How much data for training?

### ISSUE THREAD

**Issue by Samuel150** [bernard0047/LPRnet#91]
How many licence plates did you use for training or have you made transfer learning?

**Comment by bernard0047** [comment]
I'm just modifying the pipeline to work on Indian plates for later use, have used plates from [this dataset](https://www.kaggle.com/thamizhsterio/indian-license-plates) for testing things out.

**Comment by luotianhang** [comment]
i am training the model, but why the loss can not reduce

**Comment by bernard0047** [comment]
Loss starts reducing after 10-12 epochs, I trained on 120 (+30 finetuning) epochs. So I kept the  LR constant at 1e-4 for at least 80 epochs.  Use more than 5000 images in training.
